src/Controller/CommentsController.php

```diff
@@ -254,16 +255,25 @@ private function url(string $itemType, Comment $comment): ?string
      *
      * @return JsonResponse
      */
-    private function addComment(Request $request, string $itemType, $item, UserInterface $user): Response
+    private function addComment(Request $request, string $itemType, $item, UserInterface $user, $bucket): Response
     {
         if (!$user instanceof User) {
             return new JsonResponse(['error' => ['message' => 'Invalid user']], Response::HTTP_UNAUTHORIZED);
         }
 
         $parentId = $request->request->get('parent');
         $content = $request->request->get('content');
+        $fileUrl = null;
+        $fileMimeType = null;
 
-        $command = new AddCommentCommand($itemType, $item, $user, $content, (int) $parentId);
+        $file = $request->files->get('file');
+
+        if ($file->isValid()) {
```

Comment:
This should probably be checking that the comment_attachments feature is enabled before trying to upload a file.